src/components/profile-page/access-tokens/access-token-created-modal.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-FileCopyrightText: 2021 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import React from 'react'
+import { cypressId } from '../../../utils/cypress-attribute'
+import { Button, Modal } from 'react-bootstrap'
+import { Trans } from 'react-i18next'
+import { CopyableField } from '../../common/copyable/copyable-field/copyable-field'
+import type { CommonModalProps } from '../../common/modals/common-modal'
+import { CommonModal } from '../../common/modals/common-modal'
+import type { AccessTokenWithSecret } from '../../../api/tokens/types'
+
+export interface AccessTokenCreatedModalProps extends Pick<CommonModalProps, 'show' | 'onHide'> {
```

Comment:
Redundant use of `Pick<CommonModalProps, 'show' | 'onHide'>`. Maybe we should split the props and/or rename them? There seems to be a use case for Common-Modals that control their own content.

Reply:
see above